process/src/GameObject.ts

```diff
@@ -458,6 +462,13 @@ class GameObject {
       return 0;
   }
 
+  tapoutLimit(tapoutData: number[]) {
+    const mode = tapoutData[0];
+    if (mode == 0) return tapoutData.length > 3 ? tapoutData[3] : 0;
+    if (mode == 2) return tapoutData.length > 5 ? tapoutData[5] : 0;
+    return 0;
+  }
+
```

Comment:
Can you elaborate to me on how `tapoutData.length` is effecting the result? Go into some detail of the mechanics of tapout.